app/javascript/app/pages/financial-resources/support-received/non-monetized-table/non-monetized-table-selectors.js

```diff
@@ -0,0 +1,31 @@
+import { createStructuredSelector, createSelector } from 'reselect';
+import { getFocus, getFocusNames } from 'utils/financial-resources';
+
+const getData = createSelector(state => state.data, data => data || null);
+const getMeta = createSelector(state => state.meta, meta => meta || null);
+
+const filterData = createSelector(getData, data => {
+  if (!data) return null;
+  return data;
+});
+
+const addFocusToData = createSelector([ filterData, getMeta ], (data, meta) => {
+  if (!data) return null;
+  const focusNames = getFocusNames(meta);
+  return data
+    .slice()
```

Comment:
I think you don't need `slice` here, the `map` function returns a new copy of an array as well.